src/generators/notebookgenerator/itemrenderer.tsx

```diff
@@ -23,8 +178,16 @@ export function notebookItemRenderer(
       let collapsed = cellRef!.model.metadata.get(
         'toc-hr-collapsed'
       ) as boolean;
-      collapsed = collapsed != undefined ? collapsed : false;
+      collapsed = collapsed !== undefined ? collapsed : false;
       cellRef!.model.metadata.set('toc-hr-collapsed', !collapsed);
+      if (cellRef && tracker) {
+        // NOTE: we can imagine a future in which this extension combines with a collapsible-header/ings extension such that we can programmatically close notebook "sections". In the meantime, we need to resort to manually "collapsing" sections...
+        if (collapsed) {
+          uncollapseCell(tracker as any, cellRef);
```

Comment:
Let's figure out the typings so we don't need this cast.

Reply:
Done.